include/aspect/newton.h

```diff
@@ -71,6 +71,22 @@ namespace aspect
        * Newton solver.
        */
       static void create_material_model_outputs(MaterialModel::MaterialModelOutputs<dim> &output);
+
+      /**
+       * Gets the Newton derivative scaling factor used for scaling the
+       * derivative part of the Newton Stokes solver in the assembly.
+       */
+      double get_newton_derivative_scaling_factor() const;
```

Comment:
Yes, that is correct. I will add that and also add a sentence that setting it to zero results into a defection correction for of the Picard iteration.